src/engine.py

```
@@ -3010,22 +3040,45 @@ async def _handle_outbound_answered(self, channel_id: str, channel: Dict[str, An
+                    caller_channel_present=channel_present,
+                )
+                return
+
+        logger.warning("Outbound AMD continueInDialplan failed", channel_id=channel_id, attempt_id=attempt_id)
+        self._outbound_awaiting_amd_channel_ids.discard(channel_id)
+        if meta:
+            await self.outbound_store.finish_attempt(attempt_id, outcome="error", error_message="continueInDialplan failed")
+            try:
+                await self.outbound_store.set_lead_state(str(meta.get("lead_id") or ""), state="failed", last_outcome="error")
+            except Exception:
+                pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)